global-shared-library/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,66 @@
+<archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0"
+                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                      xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
+                      name="Global Shared Library Example">
+    <requiredProperties>
+        <requiredProperty key="groupId">
+            <defaultValue>unused</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="package">
+            <defaultValue>org.jenkinsci.pipeline.sample</defaultValue>
```

Comment:
In `archetype.properties` you claim this is ignored. Either it _is_ ignored, in which case the `defaultValue` here should say so; or it is actually used, in which case the test instantiation should use this default.

Reply:
Indeed, I'll change the groupId to `org.sample` and make changes in the `archetype.properties`. Same for `scripted-pipeline` btw.